src/Card.jsx

```diff
@@ -2,28 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Card.scss';
 
-const Card = ({
+function Card({
```

Comment:
I don't like this fix.  `eslint --fix` should not be swapping function definition style for me.  😢 

I will need to address this.